src/fieldnorm/writer.rs

```diff
@@ -6,13 +6,20 @@ use schema::Field;
 use schema::Schema;
 use std::io;
 
+/// The `FieldNormsWriter` is in charge of tracking the fieldnorm byte
+/// of each document for each field with field norms.
+///
+/// `FieldNormsWriter` stores a Vec<u8> for each tracked field, using a
```

Comment:
I totally missed that. I'm not fond at all of the `Vec<Vec<u8>>`. We should probably open another issue.